src/main/java/ch/njol/skript/hooks/spark/SparkHook.java

```diff
@@ -0,0 +1,34 @@
+package ch.njol.skript.hooks.spark;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.hooks.Hook;
+import me.lucko.spark.api.Spark;
+import me.lucko.spark.api.SparkProvider;
+import org.bukkit.plugin.Plugin;
+
+import java.io.IOException;
+
+public class SparkHook extends Hook<Plugin> {
+
+	public SparkHook() throws IOException {}
+
+	public static Spark spark;
+
+	@Override
+	protected boolean init() {
+		spark = SparkProvider.get();
+		return spark != null;
+	}
+
+	@Override
+	protected void loadClasses() throws IOException {
+		if (spark != null) {
+			Skript.getAddonInstance().loadClasses(getClass().getPackage().getName() + ".spark");
```

Comment:
i'm nearly certain this is the wrong package name. This will load `ch.njol.skript.hooks.spark.spark`.